### ISSUE TITLE
Can not create chart when "Series limit" set

### ISSUE THREAD

**Issue by iercan** [apache/superset#13693]
I'm trying to create line chart (same thing happen for all charts). Looks like superset can not create query properly and puts NULL for all series values. 

Here is how I configure chart 

![image](https://user-images.githubusercontent.com/3406152/111642341-100b6000-880f-11eb-9ccb-18c3dcee69fd.png)

Here is how superset generate query

```
SELECT "duration" AS "duration",
       COUNT(*) AS "count"
FROM "druid"."mydatasource"
WHERE "__time" >= '2021-03-11 14:24:38.000000'
  AND "__time" < '2021-03-18 14:24:38.000000'
GROUP BY "duration"
ORDER BY "count" DESC
LIMIT 5;

SELECT "duration" AS "duration",
       TIME_FLOOR("__time", 'PT5M') AS "__timestamp",
       COUNT(*) AS "count"
FROM "druid"."mydatasource"
WHERE "__time" >= '2021-03-11 14:24:38.000000'
  AND "__time" < '2021-03-18 14:24:38.000000'
  AND ("duration" = NULL
       OR "duration" = NULL
       OR "duration" = NULL
       OR "duration" = NULL
       OR "duration" = NULL)
GROUP BY "duration",
         TIME_FLOOR("__time", 'PT5M')
LIMIT 10000;
```

And this is actual result of first select query on druid 

![image](https://user-images.githubusercontent.com/3406152/111642269-fc5ff980-880e-11eb-9633-058d009d6b87.png)

### Expected results

Chart should be generated for selected series

### Actual results

No data return

### Environment

- superset version: 1.0.1
- python version: 3.7
- druid: 0.20.1

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by junlincc** [comment]
are you one latest master or 1.0.1? @iercan

**Comment by iercan** [comment]
@junlincc 1.0.1

**Comment by iercan** [comment]
Problem seems resolved on 1.5.0
